docs/readthedocs/usersguide.md

```diff
@@ -30,28 +30,27 @@ You can find and download the source code of the latest stable SIMONA version [h
 Docker is a tool used to containerize the entire setup. A container is like a lightweight, isolated environment that bundles everything the software needs into a single package. It is the tool used to handle the entire project setup in a single space. 
 
 Using Docker, various applications across different platforms can be run simultaneously. Instead of installing different plugins required for simona on the local machine, docker can be used to run the entire simona setup after creating an image while interacting with different environments.
-Each component has its own environment inside the Docker container setup and are defined in a yml file to ensure consistency.
+Each component has its own environment inside the Docker container setup and are defined to ensure consistency.
```

Comment:
Sorry, this still doesn't make sense for the reasons I stated earlier. Please rephrase